plugins/inputs/logfile/tailersrc.go

```diff
@@ -76,6 +76,8 @@ type tailerSrc struct {
 	startTailerOnce sync.Once
 	cleanUpFns      []func()
 }
+// Verify tailerSrc implements LogSrc
+var _ logs.LogSrc = (*tailerSrc)(nil)
```

Comment:
These are optional changes.
I noticed tailerSrc did not actually implement all methods of the interface because some methods used pointer-receiver, others used value-receiver.

https://go.dev/tour/methods/8
says:
>In general, all methods on a given type should have either value or pointer receivers, but not a mixture of both.